From 29deba7cdcbec627151de6482d6d78af55da4dfb Mon Sep 17 00:00:00 2001
From: Kamil Dudka <kdudka@redhat.com>
Date: Wed, 23 May 2018 14:34:08 +0200
Subject: [PATCH] winbuild.py: use NotImplementedError instead of
 NotImplemented

---
 winbuild.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/winbuild.py b/winbuild.py
index 883255c..7e03290 100644
--- a/winbuild.py
+++ b/winbuild.py
@@ -337,7 +337,7 @@ class OpensslBuilder(Builder):
 
     @property
     def dll_paths(self):
-        raise NotImplemented
+        raise NotImplementedError
 
     @property
     def include_path(self):
@@ -371,7 +371,7 @@ class CaresBuilder(Builder):
 
     @property
     def dll_paths(self):
-        raise NotImplemented
+        raise NotImplementedError
 
     @property
     def include_path(self):
@@ -431,7 +431,7 @@ BUILD_STATIC_LIB=1
 
     @property
     def dll_paths(self):
-        raise NotImplemented
+        raise NotImplementedError
 
     @property
     def include_path(self):
-- 
2.14.3

