From 0941cfa2dcc264ddd66a2b2f5212819bb3d2e538 Mon Sep 17 00:00:00 2001
From: Kamil Dudka <kdudka@redhat.com>
Date: Tue, 5 Jan 2016 22:40:32 -0500
Subject: [PATCH] COPYING-LGPL: update FSF address

Upstream-commit: 8304b49390c004c516f2c9511c8e68351a775404
Signed-off-by: Kamil Dudka <kdudka@redhat.com>
---
 COPYING-LGPL | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/COPYING-LGPL b/COPYING-LGPL
index 99dce33..4362b49 100644
--- a/COPYING-LGPL
+++ b/COPYING-LGPL
@@ -2,7 +2,7 @@
                        Version 2.1, February 1999
 
  Copyright (C) 1991, 1999 Free Software Foundation, Inc.
-     59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
 
@@ -485,7 +485,7 @@ convey the exclusion of warranty; and each file should have at least the
 
     You should have received a copy of the GNU Lesser General Public
     License along with this library; if not, write to the Free Software
-    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
 Also add information on how to contact you by electronic and paper mail.
 
@@ -500,5 +500,3 @@ necessary.  Here is a sample; alter the names:
   Ty Coon, President of Vice
 
 That's all there is to it!
-
-
-- 
2.5.0

